packages/frontend/core/src/desktop/pages/workspace/all-page/all-page.tsx

```diff
@@ -123,18 +135,29 @@ export const AllPage = () => {
       : null
   );
 
+  useEffect(() => {
```

Comment:
these two effects do not need to be wrapped in useEffect

Reply:
setState can happened in renderer, but in this case, useEffect may reduce the condition checks. so forget about it